Rakefile

```diff
@@ -217,6 +217,33 @@ namespace :release do
        "Groonga #{version}!!!")
     sh("git", "push", "origin", "v#{version}")
   end
+
+  desc "Post release announces"
+  task :announce do
+    latest_news = Dir.glob("doc/source/news/*.*").max do |a, b|
+      File.basename(a).to_f - File.basename(b).to_f
```

Comment:
It seems that `to_i` is enough.

Reply:
fix: [f09a27f](https://github.com/groonga/groonga/pull/2291/commits/f09a27f0bcbd0dbb7726ac2bca208e96fba365a5)